src/App.tsx

```diff
@@ -21,14 +41,27 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading
+                ? <Loader />
+                : (
+                  <>
+                    {!!allTodos.length && !isLoading && (
```

Comment:
```suggestion
                    {!!allTodos.length && (
```
це лежить в тернарці, тут і так буде !isLoading